docs/reference/announcements/880.md

```diff
@@ -115,6 +115,15 @@ Camunda 8.8 introduces a streamlined architecture, consolidating core components
 
 You can download the alpha release of the unified package from the Camunda GitHub repository, either as an executable Java application (Camunda Orchestration Core) or a Docker image.
 
+### Identity and authentication limitations in 8.8.0-alpha2
+
+The 13.0.0-alpha2 Helm chart released with Camunda 8.8.0-alpha2 establishes a new default setup to support 8.8 [Identity management updates](#identity-management-updates-saasself-managed). Currently, this setup is limited to the following components:
+
+- The Orchestration core (Zeebe, Operate, Tasklist, and Orchestration Identity)
```

Comment:
We don't have "Orchestration Identity" elsewhere in the docs -- do we need to newly define this?

Reply:
Part of https://github.com/camunda/camunda-docs/pull/5133, so this could be linked if needed once that's merged.